powerline/segments/vim/plugin/ctrlp.py

```diff
@@ -0,0 +1,102 @@
+# vim:fileencoding=utf-8:noet
+from __future__ import (unicode_literals, division, absolute_import, print_function)
+
+try:
+	import vim
+except ImportError:
+	vim = object()
+
+from powerline.bindings.vim import create_ruby_dpowerline
+
+
+def initialize():
+	global initialized
+	if initialized:
+		return
+	initialized = True
+	vim.command('echom 3210')
+	vim.command('let g:ctrlp_status = {}')
```

Comment:
Since this one is powerline-specific and not ctrlp variable it needs to be `g:_powerline_ctrlp_status`. `g:ctrlp_status` looks like a variable that is defined by or for ctrlp.